modules/ROOT/partials/configuration/text_patterns_lookup.adoc

```diff
@@ -0,0 +1,33 @@
+[[text_patterns_lookup]]
+== `+text_patterns_lookup+`
+
+This option allows you to specify a function that dynamically append additional text patterns to the xref:content-behavior-options.adoc#text_patterns[text_patterns]. The function takes a context object as an argument and should return a list of text patterns. The function will be called whenever the editor is attempting to do text pattern matching.
```

Comment:
No `you`, you shouldn't use pronouns per https://github.com/tinymce/tinymce-docs/blob/staging/docs-6/CONTRIBUTING.md#pronouns